src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/SeparatorWrapCheckTest.java

```diff
@@ -60,7 +60,7 @@ public void testDot()
         final String[] expected = {
             "31:10: " + getCheckMessage(MSG_LINE_NEW, "."),
         };
-        verify(checkConfig, getPath("InputSeparatorWrap.java"), expected);
+        verify(checkConfig, getPath("InputSeparatorWrapForTestDot.java"), expected);
```

Comment:
InputSeparatorWrapForTestDot does not need to contain examples for COMMA. Only for DOTS as specified in tokens set.